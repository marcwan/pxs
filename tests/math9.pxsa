MODULE_BODY:
        ADDVAR          ありがとう
        ADD             8e2, -5e-3, ありがとう
        DEBUGPRINT      ありがとう
LABEL1:
        EXIT            0
END_MODULE_BODY
