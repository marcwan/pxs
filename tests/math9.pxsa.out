DEBUGPRINT: ありがとう = 799.995000
